fpgalib/adc.py

```diff
@@ -955,7 +955,7 @@ def registerReadback(self):
         @inlineCallbacks
         def func():
             # build registry packet
-            regs = self.regRun(self.RUN_MODE_REGISTER_READBACK, 0) # 0 reps?
+            regs = self.regRun(self.RUN_MODE_REGISTER_READBACK, {}, 0)
```

Comment:
Naming of the params is a good idea here but this should be done in a consistent way across the module. Should I add a commit that adds param names `info` and `reps` to **all** `self.regRun()` calls in the module?